lib/ActivityStreams.js

```diff
@@ -75,6 +80,27 @@ ActivityStreams.prototype = {
   },
 
   /**
+   * Every 6 hours pro-actively prime our emebedly cache with a normal content page load
+   */
+  _primeCache() {
+    setTimeout(() => {
+      PlacesProvider.links.getTopFrecentSites.then(links => {
+        PreviewProvider.preview.primeCache(links);
+      });
+      PlacesProvider.links.getRecentBookmarks.then(links => {
+        PreviewProvider.preview.primeCache(links);
+      });
+      PlacesProvider.links.getRecentLinks.then(links => {
+        PreviewProvider.preview.primeCache(links);
+      });
+      PlacesProvider.links.getFrecentLinks.then(links => {
+        PreviewProvider.preview.primeCache(links);
+      });
+      this._primeCache();
+    }, PRIME_CACHE_TIMEOUT);
```

Comment:
it's currently called in ActivityStream's constructor, right after the cache is cleaned here: https://github.com/mozilla/activity-streams/pull/304/files#diff-cd9b1d133214f36f1c20099d7f6817e6R52 
is that what you mean?